src/data_types/mod.rs

```diff
@@ -39,7 +39,7 @@ impl Handle {
 
 /// Handle to an event structure
```

Comment:
I think the docstring could use an update, modeled after https://github.com/rust-osdev/uefi-rs/blob/9833ee17e09abdc132899a5d4c161979f5f0f54e/src/data_types/mod.rs#L7 (guaranteed to be nonnull, use `Option<Event>` if you need to handle nullability)